.env.template

```diff
@@ -31,16 +31,16 @@ POSTGRESQL_HOST=localhost
 POSTGRESQL_PORT=5432
 POSTGRESQL_USER_SUFFIX=
 PGSSLMODE=disable
-POSTGRESQL_ROOT=postgres
-POSTGRESQL_ROOT_PASSWORD=postgres
+POSTGRESQL_ROOT=root
+POSTGRESQL_ROOT_PASSWORD=root
 
 # pgAdmin
 PGADMIN_PORT=9999
 
 # RabbitMQ Connection
 RABBITMQ_PROTOCOL=amqps
-RABBITMQ_HOST=rabbitmq.eu.axinom.net
+RABBITMQ_HOST=rabbitmq.test.axinom.net
```

Comment:
The template should ideally not be changed. we should continue using prod urls and I would prefer to keep old credentials, otherwise, there is probably a need to fully reset dev postgres server...

Reply:
@AxTrusov Why are we running tests against prod?

Reply:
we do not, the message-sending parts are mocked in tests